README.md

```diff
@@ -432,38 +420,6 @@ in the following order, with optional plugin provided scripts in bold:
 1. **post_docs_test.sh**
 1. **post_script.sh**
 
-The pipeline can be modified, see the help text for available options.
-
-```
-$ ./plugin-template --help
-usage: plugin-template [-h] [--generate-config] [--bootstrap] [--test]
-                       [--travis] [--docs] [--all] [--verbose]
-                       plugin_name plugin_app_label
-
-Generate a .travis.yml and .travis directoryfor a specified plugin
-
-positional arguments:
-  plugin_name           Create or update this plugin.
-  plugin_app_label      the Django app label for the plugin - usually the part after pulp_ or
-                     pulp-
-
-
-optional arguments:
-  -h, --help            show this help message and exit
-  --bootstrap           Create a new plugin and template boilerplate code.
-
-  --test                Generate or update functional and unit tests.
-
-  --travis              Generate or update travis configuration files.
-
-  --docs                Generate or update plugin documentation.
-
-  --all                 Create a new plugin and template all non-excluded files.
-
-  --verbose             Include more output.
-
-```
```

Comment:
why removing it?

Reply:
Trying to remember, why:
It is outdated, and the information here is doubled anyway. It looked to me like a stop gap until better content was written.